### PR TITLE
Updates and Queries

### PR DESCRIPTION
**Pull request by gracekabaghe**
- Added a column species of type string to the animals' table.
- Inserted additional data
- deleted and rollback records
- Ran some queries

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -18,4 +18,50 @@ VALUES ('Pikachu', '01-07-2021', '1', 'false', '15.08');
...
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-1-1';
+SAVEPOINT delete_one;
+UPDATE animals SET weight_kg = -1 * weight_kg;
+ROLLBACK TO delete_one;
+UPDATE animals SET weight_kg = -1 * weight_kg WHERE weight_kg < 0;
+COMMIT;
```
Comment (rukundoeric) [inline data.sql]: - [x] <b>REQUIRED:</b> These should be moved to `queries.sql` as the project requires so, and Kindly make sure to verify that changes have been made before and after `COMMIT`. something like this:

```sql
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @gracekabaghe, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 1 above)
Comment (gracekabaghe) [reply data.sql]: moved to queries.sql

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉✔🍾**
Hello @gracekabaghe 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**